src/hooks/useSaveVideoFrame.ts

```diff
@@ -0,0 +1,54 @@
+import { RefObject, useEffect } from 'react'
```

Comment:
if we arent doing the save here can we remove this, just want to not inject bugs, not saying its buggy just would rather it be in once we get the current feature set working well

Reply:
Yeah totally I'll just drop it!